### ISSUE TITLE
Red outage are drawn above legend box 

### ISSUE THREAD

**Issue by lightonflux** [phpservermon/phpservermon#228]
Version 3.1.1.

Red outage are drawn above legend box:

![screenshot of Red outage are drawn above legend box ](https://cloud.githubusercontent.com/assets/1377943/10255490/098d632e-694b-11e5-85dd-f0ecb71484ef.png)

**Comment by sfennell** [comment]
I resolved the issue by changing the color of the outage rectangles and using postDrawHooks in static/js/history.js

```patch
97c97
<                               color: '#fe5d5d',
---
>                               color: 'rgba(200, 0, 0, 0.3)',
162a163,170
>
>         $.jqplot.postDrawHooks.push(function() {
>           var overlayCanvas = $($('.jqplot-overlayCanvas-canvas')[0]);
>           var legend = $($('.jqplot-table-legend')[0]);
>           legend.detach();
>           overlayCanvas.after(legend);
>         });
>

**Comment by sadortun** [comment]
Thanks @sfennell  !

![image](https://cloud.githubusercontent.com/assets/794449/15412927/ce6eb798-1df9-11e6-933f-b206f3ddf421.png)
